app/assets/stylesheets/components/_alert.scss

```diff
@@ -0,0 +1,18 @@
+.irs-reproofing-notice {
+  background-color: #007187;
+  color: #ffffff;
+}
+
+.irs-reproofing-notice:before {
+  background-image: url('alert/irs-info.svg') no-repeat center/contain;
+  background-color: #fff;
+}
+
+@supports ((-webkit-mask: url('')) or (mask: url(''))) {
+  .irs-reproofing-notice:before {
+    background: none;
+    background-color: #fff;
+    -webkit-mask: url('alert/irs-info.svg') no-repeat center/contain;
+    mask: url('alert/irs-info.svg') no-repeat center/contain;
+  }
+}
```

Comment:
Couple thoughts:

- We can export the icon SVG with the white background to avoid dealing with the mask property and related supports
- Since the file name implies this applies generalized alert styles, I think we could rename the class to avoid such a strong association to the IRS use-case
- We should use [color tokens](https://design.login.gov/utilities/color/) when available
- The `background-image` value here is invalid, since it should only contain the `url` portion

Icon:

```svg
<svg width="16" height="24" viewBox="0 0 16 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M14.927 7.984a7.966 7.966 0 0 0-2.911-2.911A7.825 7.825 0 0 0 8 4c-1.451 0-2.79.358-4.015 1.073a7.963 7.963 0 0 0-2.912 2.911A7.825 7.825 0 0 0 0 12c0 1.451.358 2.79 1.073 4.016a7.965 7.965 0 0 0 2.911 2.911A7.825 7.825 0 0 0 8 20c1.452 0 2.79-.358 4.016-1.073a7.965 7.965 0 0 0 2.911-2.911A7.827 7.827 0 0 0 16 12c0-1.451-.358-2.79-1.073-4.016ZM6.667 6c0-.097.031-.177.094-.24A.325.325 0 0 1 7 5.667h2c.097 0 .176.03.239.093a.325.325 0 0 1 .094.24v1.667a.325.325 0 0 1-.094.24A.323.323 0 0 1 9 8H7a.324.324 0 0 1-.24-.094.324.324 0 0 1-.093-.24V6Zm4 11a.325.325 0 0 1-.094.24.324.324 0 0 1-.24.093H5.668a.324.324 0 0 1-.24-.094.324.324 0 0 1-.094-.24v-1.666c0-.097.032-.177.094-.24a.325.325 0 0 1 .24-.093h1v-3.333h-1a.324.324 0 0 1-.24-.094.324.324 0 0 1-.094-.24V9.667c0-.098.032-.177.094-.24a.324.324 0 0 1 .24-.094H9c.098 0 .177.032.24.094a.325.325 0 0 1 .094.24V15h1c.097 0 .177.031.239.094a.325.325 0 0 1 .094.24V17Z" fill="#fff"/></svg>
```

```suggestion
.usa-alert--info-important {
  @include u-bg('accent-cool-darker');
  color: color('white');
  
  &::before {
    background-image: url('alert/info-white.svg');
  }
}
```